ironfish-cli/src/snapshot.ts

```diff
@@ -37,11 +37,8 @@ export const getDefaultManifestUrl = (networkId: number): string | null => {
 const tryParseUrl = (url: string): URL | null => {
   try {
     return new URL(url)
```

Comment:
This is the only real change right?

Reply:
There's one more fix to how `totalEntries` is incremented aside from this one.